internal/registry/registry.go

```diff
@@ -0,0 +1,21 @@
+package registry
```

Comment:
Maybe I can rename this directory from "registry" to "operator-registry" to make it even more transparent? Or add a package godoc comment that explains these types were manually vendored from that repository if it makes it easier for our future selves?

Reply:
Updated in the last force push.